src/lib/test-handler.js

```diff
@@ -115,13 +106,28 @@ export default class TestHandler {
           console.log(this.outputHandler.getSummaryOutput());
         }
       }
-    });
 
-    worker.on('error', (worker, error) => {
-      console.log('Error caught: ', error);
-    });
+      if (payload.exception) {
+        console.log('Error caught: ', payload.exception);
+      }
+    }.bind(this);
```

Comment:
Can we swap these function calls in the PR to use arrow notation? So instead of `let done = function(payload) {` we do `let done = (payload) => {`. Arrow functions don't change scope, so you won't need to `.bind(this)` anywhere, and it would be better for consistency with the rest of the codebase